### ISSUE TITLE
After providing liquidity my pool share % when to "NaN"

### ISSUE THREAD

**Issue by lauracw100** [scrtlabs/EthereumBridgeFrontend#142]
Sorry to be a pain, but I bugs me when I see "NaN" is displayed in such a nice UI :-P.

![image](https://user-images.githubusercontent.com/436905/115612824-aec92780-a2b9-11eb-8771-92eed9bddc6c.png)

Right after the "Provide" transaction completed, the UI updated and that's when I noticed the NaN % of pool share.

**Comment by cankisagun** [comment]
is this a refresh issue @Cashmaney

**Comment by alejandrordzl** [comment]
We haven't touched that part of the code. I reviewed the variable that's been displayed though and noticed an expected Big Number is being returned as null from the backend.

![2021-04-26_12h36_47](https://user-images.githubusercontent.com/81386118/116141532-aa23bb00-a68d-11eb-87a0-ff2c42d9c6fb.png)

We can 2 two things:
 * Default value if null
 * Hide the % if null

**Comment by cankisagun** [comment]
can we just show "-"

**Comment by lauracw100** [comment]
I can confirm I'm now seeing "-" for the value of "Your Current Share of Pool." But I was expecting to see some value > 0. 

This may become a consistent support issue from users if they don't get the UI feedback on their current share of the pool.

**Comment by cankisagun** [comment]
My bad - I thought we were getting `NaN` when there's no liquidity. @lauraweindorf can you pls confirm you have liquidity in the pair (that's not staked in an earn pool) and you are still seeing "-"?

**Comment by lauracw100** [comment]
no worries @cankisagun! 

Okay, yeah so here's a snapshot just before I'm providing liquidity for the SEFI/sSCRT pair:

![image](https://user-images.githubusercontent.com/436905/116306730-70c57b00-a773-11eb-9515-bf22c35d1bef.png)

This is the successful screen, where I'm expecting to see my percent of the pool to be > 0:

![image](https://user-images.githubusercontent.com/436905/116306893-a4080a00-a773-11eb-811c-83c54571db8e.png)

Maybe I'm misunderstanding something though?

I reloaded the "Provide" page to see if it was a refresh issue, but it still says my share of the pool is "-".

And then when I go to the "Withdraw" page to see what that says my total pooled tokens are >244 yet the # of tokens for each is 0, and my pool share is also showing 0%.

![image](https://user-images.githubusercontent.com/436905/116307296-242e6f80-a774-11eb-8e3f-8b21ef64b5d4.png)

**Comment by cankisagun** [comment]
@David-Sod these seem to be working on the current swap UI - I think you guys must be ommitting something

![image](https://user-images.githubusercontent.com/15679491/116314025-6e5b2500-a763-11eb-9c44-b860ffedfab1.png)

![image](https://user-images.githubusercontent.com/15679491/116314107-88950300-a763-11eb-8d00-ec3eaaf8084a.png)

We gotta get the actual numbers "-" is not going to work. I misread the prev. issue

Can you pls check with @Cashmaney

**Comment by alejandrordzl** [comment]
Yeah, when we separated the components and during the process forget a function, now this issues should be fixed.
